source/Ship.cpp

```diff
@@ -2364,6 +2364,9 @@ int Ship::Scan()
 	// Play the scanning sound if the actor or the target is the player's ship.
 	if(isYours || (target->isYours && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
+	// Potential relation penalties for scanning.
+	if(isYours && target->GetGovernment() && (result & ShipEvent::SCAN_OUTFITS || result & ShipEvent::SCAN_CARGO))
+		target->GetGovernment()->Offend(ShipEvent::SCAN_OUTFITS, target->CrewValue());
```

Comment:
Would we want to split outfit scanning and cargo scanning into two different offenses?

Reply:
I asked Zitchas and he said he didnt want to

Reply:
My position is mostly that, given that the two forms of scanning are tied together everywhere else, they might as well stay unified here too. And as far as that goes, I don't think there's any significant gameplay experience to be gained by splitting them apart.

Reply:
actually now that I think about it in correct OO "result" would have to be passed trough this function not always scan outfit and then the other function will decide if it wants to do the same for both (same final result but different manner)
I'm just going to stock both ship events with the same value from "scanning"